### ISSUE TITLE
Usage of "numbering-start-at" unclear. 

### ISSUE THREAD

**Issue by saschakr** [asciidoctor/asciidoctor-pdf#1710]
Hello everybody,

I've waited for [Pull#1649](https://github.com/asciidoctor/asciidoctor-pdf/pull/1649) because I need to start my page numbering on page 2 (with number 2), but this is a little unclear to me.

I've made following example.

**page.adoc**
```
= Testpage
:toc: macro
:sectnums:

<<<
toc::[]
<<<
== page one
<<<
== page two
<<<
== page three
```

**theme.yml**
```
extends: default

running-content:
  start-at: 2

page:
  numbering-start-at: 2

header:
  height: 0.75in
  line-height: 1.2
  recto:
    center:
      content: '{page-number}'
  verso:
    center:
      content: '{page-number}'
```

I would except my page numbering starts on page 2 with number 2 but it is still on page 1.
It would work if I set the `:title-page: true` within the adoc file, but I need to customize it too much.

So my question is: What am I doing wrong? Did I misunderstood the pull  #request?

EDIT:

```
asciidoctor-pdf --version
Asciidoctor PDF 1.5.3 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
```

**Comment by mojavelinux** [comment]
That change is not yet available in a release. You'll either need to wait for Asciidoctor PDF 2 to be released or make the case that it should be backported to 1.5.x and released as part of a 1.5.4 release.

The docs that go with the release are here: https://github.com/asciidoctor/asciidoctor-pdf/blob/v1.5.x/docs/theming-guide.adoc#key-prefix-page

In the note, it clearly states that the body offset only works if the document has a title page. In Asciidoctor PDF 2, it will work even if the document does not have a title page.

**Comment by saschakr** [comment]
I thought it was released.
I checked out master and build it like descripted in https://github.com/asciidoctor/asciidoctor-pdf/blob/master/CONTRIBUTING-CODE.adoc

```
$ bundle --path=.bundle/gems
$ bundle exec rake install
$ gem install pkg/*.gem
```

Thanks for help!
I close the issue now.
